cmd/uplink/Dockerfile

```diff
@@ -9,6 +9,7 @@ ENV CONF_PATH=/root/.local/storj/uplink \
     SATELLITE_ADDR=
```

Comment:
@egonelbre We should check, if those env variables are still used/valid.

Reply:
no clue